### ISSUE TITLE
M1 Mac cannot run zellij.

### ISSUE THREAD

**Issue by blackcater** [zellij-org/zellij#508]
**Basic information**

`zellij --version`: v0.10.0
`tput lines`: 28
`tput cols`: 238
`uname -av` or `ver`(Windows): Darwin ElondeMac-mini.local 20.4.0 Darwin Kernel Version 20.4.0: Thu Apr 22 21:46:41 PDT 2021; root:xnu-7195.101.2~1/RELEASE_ARM64_T8101 arm64

**Further information**

When I run `zellij` in terminal:

```txt
Originating Thread(s):
1. ipc_server: NewClient
2. ipc_server: NewClient

Error: thread 'server_thread' panicked at 'cannot find layout /Users/blackcater/Library/Application Support/org.Zellij-Contributors.Zellij/layouts/default.yaml': /Users/blackcater/.cargo/registry/src/github.com-1ecc6299db9ec823/zellij-0.10.0/src/client/layout.rs:197
   0: backtrace::backtrace::trace
   1: backtrace::capture::Backtrace::new
   2: zellij::common::errors::handle_panic
   3: std::panicking::rust_panic_with_hook
   4: std::panicking::begin_panic_handler::{{closure}}
   5: std::sys_common::backtrace::__rust_end_short_backtrace
   6: _rust_begin_unwind
   7: std::panicking::begin_panic_fmt
   8: zellij::client::layout::Layout::new::{{closure}}
   9: zellij::client::layout::Layout::new
  10: zellij::server::init_session
  11: std::sys_common::backtrace::__rust_begin_short_backtrace
  12: core::ops::function::FnOnce::call_once{{vtable.shim}}
  13: std::sys::unix::thread::Thread::new::thread_start
  14: _pthread_kill
```

**Comment by a-kenji** [comment]
Thank you for taking the time filing this issue!
Did you install zellij from crates.io?

**Comment by a-kenji** [comment]
@blackcater 
I hope that the problem is resolved as of the current main, as well as `0.11.0`, would you be willing to test that out? I am closing this for now. Please do tell us if the problem still persists for you - thanks for reaching out.

**Comment by enckse** [comment]
I installed `v0.10.0` via macports and hit the above error, then uninstalled and install via `cargo install zellij` (`v0.11.0`) and it started successfully!
